src/_help/specification-support/openapi-support.md

```diff
@@ -36,3 +36,7 @@ JSON Schema provides the possibility to declare a property as read or write only
 ## Webhooks support
 
 You can use the `webhooks` field (introduced in OpenAPI 3.1) to define the API webhook payloads. Please read to the [dedicated documentation page](/help/specification-support/openapi-support/webhooks/) for more information.
+
+## Expose your beta features (`x-beta`)
+
+This custom property allows you to identify some components of your documentation as beta. Find out more in our [dedicated section](/help/specification-support/doc-beta).
```

Comment:
I've moved to the new file (cf https://github.com/bump-sh/docs/pull/148/files#r1425050816), this can be removed

```suggestion
```